<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><!-- InstanceBegin template="/Templates/moban_recomb.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<!-- InstanceBeginEditable name="doctitle" -->
<title>RECOMB 2013 Beijing, China</title>
<!-- InstanceEndEditable -->
<style type="text/css">
<!--
body,td,th {
	font-family: calibri;
}
body {
	margin-left: 0px;
	margin-top: 0px;
	margin-right: 0px;
	margin-bottom: 0px;
}
.small_text {	font-family: calibri;
	font-size: 12px;
	line-height: normal;
	font-weight: normal;
	font-variant: normal;
	color: #333;
}
.body_text {
	font-family: calibri;
	font-size: 15px;
	font-style: normal;
	font-weight: normal;
	font-variant: normal;
	color: #232323;
}
.sub_topic {
	font-family: calibri;
	font-size: 20px;
	font-style: normal;
	font-weight: normal;
	text-transform: uppercase;
	color: #400040;
}
.small_title {	font-family: calibri;
	font-size: 16px;
	font-style: normal;
	line-height: normal;
	font-weight: normal;
	font-variant: normal;
	text-transform: uppercase;
	color: #EF8201;
	background-color: #FFF;
	height: auto;
	width: auto;
}
.body_text_bold {
	font-family: calibri;
	font-size: 15px;
	font-style: normal;
	font-weight: bold;
	font-variant: normal;
	color: #232323;
}
.body_text_bold_white {
	font-family: calibri;
	font-size: 15px;
	font-style: normal;
	font-weight: bold;
	font-variant: normal;
	color: #FFFFFF;
}
.body_text_italic {
	font-family: calibri;
	font-size: 16px;
	font-style: italic;
	font-weight: normal;
	font-variant: normal;
	color: #232323;
}
.small_text_italic {
	font-family: calibri;
	font-size: 12px;
	line-height: normal;
	font-weight: normal;
	font-variant: normal;
	color: #333;
	font-style: italic;
}
.small_text_italic_red {
	font-family: calibri;
	font-size: 12px;
	line-height: normal;
	font-weight: normal;
	font-variant: normal;
	color: #990000;
	font-style: italic;
}
.small_text_red {
font-family: calibri;
	font-size: 12px;
	line-height: normal;
	font-weight: normal;
	font-variant: normal;
	color: #990000;
}
-->
</style>
<!-- InstanceBeginEditable name="head" -->
<!-- InstanceEndEditable -->
</head>

<body>
<table width="1024" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
    <td colspan="2">&nbsp;</td>
    <td>&nbsp;</td>
  </tr>
  <tr>
    <td width="12" background="pic/backcover_left.jpg">&nbsp;</td>
    <td width="1000" height="187" colspan="2" align="left" valign="top"><img src="pic/topbanner.jpg" alt="Top Banner" width="1000" height="187" border="0" align="top" usemap="#Map" /></td>
    <td width="12" background="pic/backcover_right.jpg">&nbsp;</td>
  </tr>
  <tr>
    <td width="12" background="pic/backcover_left.jpg" ><p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p></td>
    <td width="206" align="left" valign="top" bgcolor="#EECCED" ><img src="pic/menu.jpg" alt="menu" width="206" height="980" border="0" usemap="#Map2" /></td>
    <td width="794" align="center" valign="top" bgcolor="#FFFFFF" ><table width="90%" border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td>&nbsp;</td>
      </tr>
      <tr>
        <td height="800" align="left" valign="top"><!-- InstanceBeginEditable name="EditRegion3" -->
          <p class="sub_topic">Accepted papers</p>

        <p class="body_text">
            The conference proceedings (<i>Lecture Notes in Computer Science</i>, Volume 7821, 2013) is now online. You can find information about it at <a href="http://www.springeronline.com/978-3-642-37194-3" target="blank"> http://www.springeronline.com/978-3-642-37194-3</a> or access the online version at <a href="http://www.springerlink.com/content/978-3-642-37194-3/" target="blannk">http://www.springerlink.com/content/978-3-642-37194-3/</a>. A subscription to Springer products is required in order for you to download full text.
        </p>
        <br/>
          
        <table width="100%" border="0" align="center" cellpadding="1" cellspacing="0">
        <tr bgcolor=#E8E8E8>
        <th align="left" width="6%">ID</th>
        <th align="left">Authors and Title</th>
        </tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>2</td>
<td>Evan Senter, Saad Sheikh, Ivan Dotu, Yann Ponty and Peter Clote</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Using the Fast Fourier Transform to accelerate the computational search for RNA conformational switches</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>5</td>
<td>Dong Xu, Hua Li and Yang Zhang</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Fast and Accurate Calculation of Protein Depth by Euclidean Distance Transform</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>17</td>
<td>Mohammad Javad Sadeh, Giusi Moffa and Rainer Spang</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Considering Unknown Unknowns - Reconstruction of Non-confoundable Causal Relations in Biological Networks</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>24</td>
<td>Yufeng Wu</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> An Algorithm for Constructing Parsimonious Hybridization Networks with Multiple Phylogenetic Trees</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>30</td>
<td>Dong-Yeon Cho and Teresa Przytycka</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Dissecting cancer heterogeneity with a probabilistic genotype-phenotype model</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>31</td>
<td>Mingxun Wang and Nuno Bandeira</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Spectral Library Generating Function for Assessing Spectrum-Spectrum Match Significance</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>32</td>
<td>Zhizhuo Zhang, Guoliang Li, Kim-Chuan Toh and Wing-Kin Sung</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Inference of Spatial Organizations of Chromosomes Using Semi-definite Embedding Approach and Hi-C Data</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>35</td>
<td>Roy Ronen, Nitin Udpa, Eran Halperin and Vineet Bafna</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Learning Natural Selection from the Site Frequency Spectrum</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>42</td>
<td>Jesper Jansson, Chuanqi Shen and Wing-Kin Sung</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> An Optimal Algorithm for Building the Majority Rule Consensus Tree</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>43</td>
<td>Itamar Eskin, Farhad Hormozdiari, Lucia Conde, Chris Skibola, Jacques Riby, Eleazar Eskin and Eran Halperin</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> eALPS: Estimating Abundance Levels in Pooled Sequencing Using Available Genotyping Data</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>49</td>
<td>Kriti Puniyani and Eric Xing</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> NP-MuScL: Unsupervised global prediction of interaction networks from multiple data sources</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>53</td>
<td>Emrah Kostem and Eleazar Eskin</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Efficiently Identifying Significant Associations in Genome-wide Association Studies</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>56</td>
<td>Dan He, Zhanyong Wang, Laxmi Parida and Eleazar Eskin</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> IPED: Inheritance Path based Pedigree Reconstruction Algorithm using Genotype Data</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>62</td>
<td>Ting Wang, Michael Stevens, Jeffrey Cheng, Mingchao Xie and Joseph Costello</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> MethylCRF, an Algorithm for Estimating Absolute Methylation Levels at Single CpG Resolution from Methylation Enrichment and Restriction Enzyme Sequencing Methods </i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>72</td>
<td>Alina Munteanu and Raluca Gordan</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Distinguishing between genomic regions bound by paralogous transcription factors</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>74</td>
<td>Xiaowen Liu, Shawna Hengel, Si Wu, Nikola Tolic, Ljiljana Pasa-Tolic and Pavel Pevzner</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Identification of proteins with multiple post-translational modifications using top-down tandem mass spectra</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>75</td>
<td>Shijian Chen, Anqi Wang and Lei Li</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> SEME: A fast mapper of Illumina sequencing reads with statistical evaluation</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>80</td>
<td>Ngoc Hieu Tran, Kwok Pui Choi and Louxin Zhang</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Counting Motifs in the Entire Biological Network from Noisy and Incomplete Data</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>92</td>
<td>Ali A Faruqi, William A Bryant and John W Pinney</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Analysis of metabolic evolution in bacteria using whole-genome metabolic models</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>96</td>
<td>Kyowon Jeong, Sangtae Kim and Pavel Pevzner</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> UniNovo : a universal tool for de novo peptide sequencing</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>105</td>
<td>Sebastian Will, Christina Schmiedl, Milad Miladi, Mathias Mohl and Rolf Backofen</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> SPARSE: Quadratic Time Simultaneous Alignment and Folding of RNAs Without Sequence-Based Heuristics</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>110</td>
<td>Fei Guo, Shuai Cheng Li, Wenji Ma and Lusheng Wang</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Detecting Protein Conformational Changes in Interactions via Scaling Known Structures</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>112</td>
<td>Vladimir Reinharz, Yann Ponty and Jerume Waldispuhl</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> A linear inside-outside algorithm for correcting sequencing errors in structured RNA sequences W</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>115</td>
<td>Mukul S. Bansal, Eric J Alm and Manolis Kellis</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i>Reconciliation Revisited: Handling Multiple Optima when Reconciling with Duplication, Transfer, and Loss</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>120</td>
<td>Jesse Rodriguez, Serafim Batzoglou and Sivan Bercovici</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> An Accurate Method for Inferring Relatedness in Large Datasets of Unphased Genotypes via an Embedded Likelihood-Ratio Test</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>121</td>
<td>Fabio Vandin, Alexandra Papoutsaki, Ben Raphael and Eli Upfal</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Genome-Wide Survival Analysis of Somatic Mutations in Cancer</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>127</td>
<td>Raheleh Salari, Syed Shayon Saleh, Dorna Kashef-Haghighi, David Khavari, Daniel E Newburger, Robert E West, Arend Sidow and Serafim Batzoglou</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Inference of Tumor Phylogenies with Improved Somatic Mutation Discovery</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>133</td>
<td>Christopher Reeder and David Gifford</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> High Resolution Modeling of Chromatin Interactions</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>146</td>
<td>Layla Oesper, Ahmad Mahmoody and Ben Raphael</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Estimating Tumor Purity and Cancer Subpopulations from High-Throughput DNA Sequencing Data</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>147</td>
<td>Chittaranjan Tripathy, Anthony Yan, Pei Zhou and Bruce Donald</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Extracting Structural Information from Residual Chemical Shift Anisotropy: Analytic Solutions for Peptide Plane Orientations and Applications to Determine Protein Structure</i></td>
</tr>

<tr valign="top" bgcolor=#FFFFFF>
<td>160</td>
<td>Sergey Nurk, Anton Bankevich, Dmitry Antipov, Alexey Gurevich, Anton Korobeynikov, Alla Lapidus, Andrey Prjibelsky, Alexey Pyshkin, Alexander Sirotkin, Yakov Sirotkin, Ramunas Stepanauskas, Jeffrey McLean, Roger Lasken, Glenn Tesler, Max Alekseyev and Pavel Pevzner</td>
</tr>
<tr valign="top" bgcolor=#FFFFFF>
<td>&nbsp;</td>
<td><i> Assembling Genomes and Multigenomes from Highly Chimeric Reads</i></td>
</tr>

<tr valign="top" bgcolor=#E8E8E8>
<td>178</td>
<td>Shuigeng Zhou</td>
</tr>
<tr valign="top" bgcolor=#E8E8E8>
<td>&nbsp;</td>
<td><i> Boosting Prediction Performance of Protein-Protein Interaction Hot Spots by Using Structural Neighborhood Properties</i></td>
</tr>



        </table>  
          
        <br/>  
          
          

        <!-- InstanceEndEditable --></td>
      </tr>
    </table></td>
    <td width="12"  background="pic/backcover_right.jpg">&nbsp;</td>
  </tr>
  <tr>
    <td width="12" background="pic/backcover_left.jpg">&nbsp;</td>
    <td colspan="2"><img src="pic/bottom_banner.jpg" width="1000" height="28" alt="bottom banner" /></td>
    <td width="12"  background="pic/backcover_right.jpg">&nbsp;</td>
  </tr>
</table>

<map name="Map" id="Map">
  <area shape="rect" coords="194,163,251,184" href="index.htm" target="_self" alt="Home" />
  <area shape="rect" coords="486,163,553,185" href="program.html" target="_self" alt="Program" />
  <area shape="rect" coords="696,164,751,184" href="travel.html" target="_self" alt="Travel" />
  <area shape="rect" coords="374,162,464,185" href="registration.html" target="_self" alt="Registration" />
  <area shape="rect" coords="265,163,349,185" href="committee.html" target="_self" alt="Committees" />
  <area shape="rect" coords="913,162,985,185" href="contact.html" target="_self" alt="Contact" /><area shape="rect" coords="572,162,679,184" href="satellite.html" target="_self" alt="" />
  <area shape="rect" coords="774,164,892,184" href="accommodation.html" target="_self" alt="Accommodation" />
<area shape="rect" coords="21,139,349,155" href="http://bioinfo.au.tsinghua.edu.cn/recomb2013/index.htm" target="_blank" />
<area shape="rect" coords="372,140,525,155" href="http://www.recomb2013.org" target="_blank" />
</map>

<map name="Map2" id="Map2">
  <area shape="rect" coords="101,654,151,702" href="http://www.nsf.gov" target="_blank" />
  <area shape="rect" coords="19,751,77,793" href="http://www.intel.com" target="_blank" />
  <area shape="rect" coords="18,508,138,554" href="http://www.tsinghua.edu.cn/publish/then/index.html" target="_blank" />
  <area shape="rect" coords="20,561,136,598" href="http://www.tnlist.org.cn/pages/english.jsp" target="_blank" />
  <area shape="rect" coords="13,118,152,138" href="travel.html" target="_self" alt="Travel &amp; Visa" />
  <area shape="rect" coords="18,70,136,88" href="program.html" target="_self" alt="Program" />
  <area shape="rect" coords="20,380,152,399" href="About_recomb.html" target="_self" alt="About RECOMB" />
  <area shape="rect" coords="15,143,150,163" href="accommodation.html" target="_self" alt="Accommodation" />
  <area shape="rect" coords="23,237,126,255" href="keydates.html" target="_self" alt="Key Dates" />
  <area shape="rect" coords="18,20,165,42" href="index.htm" target="_self" alt="General Information" />
  <area shape="rect" coords="17,44,164,67" href="venue.html" target="_self" alt="Conference Venue" />
  <area shape="rect" coords="20,189,154,209" href="Keynote.html" target="_self" alt="Keynote Speakers" />
  <area shape="rect" coords="13,91,190,113" href="registration.html" target="_self" alt="Registration" />
  <area shape="rect" coords="20,423,146,447" href="committee.html" target="_self" alt="Committees" />
  <area shape="rect" coords="20,354,149,376" href="submission.html" target="_self" alt="Submission" />
  <area shape="rect" coords="22,329,161,350" href="callhigh.html" target="_self" />
  <area shape="rect" coords="18,306,150,326" href="call%20for%20poster.html" target="_self" alt="Call for Posters" />
  <area shape="rect" coords="19,283,137,303" href="call.html" target="_self" alt="Call for Papers" />
  <area shape="rect" coords="19,166,152,185" href="travel%20fellow.html" target="_self" alt="Travel Fellowships" />
  <area shape="rect" coords="20,402,154,422" href="About%20Beijing.html" target="_self" alt="About Beijing" />
  <area shape="rect" coords="21,451,141,471" href="sponsor.html" target="_self" alt="Sponsors" />
  <area shape="rect" coords="23,212,151,233" href="accepted_papers.html" target="_self" />
<area shape="rect" coords="20,475,101,494" href="contact.html" target="_self" alt="Contact" />
<area shape="rect" coords="23,650,89,698" href="http://www.nsfc.gov.cn/Portal0/default166.htm" target="_blank" />
<area shape="rect" coords="9,257,193,280" href="satellite.html" target="_self" alt="Stellite Workshop" />
<area shape="rect" coords="23,607,125,644" href="http://www.iscb.org/" target="_blank" />
<area shape="rect" coords="19,705,135,741" href="http://www.emc.com/domains/isilon/index.htm" target="_blank" />
<area shape="rect" coords="22,805,132,854" href="http://www.sugon.com" target="_blank" />
<area shape="rect" coords="21,867,129,901" href="http://www.illumina.com" target="_blank" />
<area shape="rect" coords="24,918,135,967" href="http://www.monsanto.com" target="_blank" />
</map>
</body>
<!-- InstanceEnd --></html>
